pkgs/tools/admin/cli53/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, pythonPackages, fetchurl }:
+
+buildPythonPackage rec {
+  name = "cli53-${version}";
+  namePrefix = "";
```

Comment:
`namePrefix = "";` is necessary to stop buildPythonPackage from prefixing the package's name with `python27-`.  It's a behaviour that makes sense for libraries, but not so much for standalone apps meant for direct installation.